setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = eedl
+description= Code to export arbitrarily large areas of data from Earth Engine to local storage and optionally process data locally afterward
+author = nickrsan, AdamJCrawford
+license = MIT
+platforms = unix, linux, osx, cygwin, win32
+url = https://github.com/water3d/eedl
+version = 0.1.0
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+
+[options]
+packages =
+    eedl
+install_requires =
+    gdal
+    rasterstats
+    earthengine-api
+    fiona
+    pandas
+    seaborn>=0.12.0
+    google-cloud-storage
+    setuptools
+    requests
+    pytest
+python_requires = >= 3.8
+package_dir =
```

Comment:
@AdamJCrawford I'm going to want to move the package out of the `src` subdirectory, I think, unless you have a better idea. Lots of tools work nicely with the project root configured so that we can import the package (because it lives there). We *could* configure tools to load packages in the `src` dir instead, but I don't see a justification for that right now. Do you know if any best practices or anything else suggest not keeping the package in the root? Otherwise, I'm inclined to relocate it.